app/partials/project-card.cjsx

```diff
@@ -12,6 +12,9 @@ counterpart.registerTranslations 'en',
 module.exports = React.createClass
   displayName: 'ProjectCard'
 
+  projectOwner: ->
+    apiClient.type(@props.project.links.owner.type).get(@props.project.links.owner.id)
```

Comment:
Is this necessary? With the include above, this resource should already be available without a new request. Accessing `links` directly makes things more fragile, since links can be defined a bunch of different ways.